### ISSUE TITLE
Pagoda 2 rev B 

### ISSUE THREAD

**Issue by Tomkanovik** [fishpepper/pagoda-antenna#3]
Hello can you make panelized gerber file with pagoda 2 rev B, please?

**Comment by john4all** [comment]
Hi Tomkanovik.  I posted a reply to this on the commit comments section yesterday.  Click over there and you should see my files for the 2b rev.

**Comment by rodgerramjet** [comment]
@john4all Sorry, where?

**Comment by zenewian** [comment]
So, is there 16x16 file for pagoda 2 rev b

**Comment by john4all** [comment]
Yes, I made one a while back and have been using the resulting antennas for the past few months with no issues.  Just never got around to adding the files to a pull request on here.  Here is the file:

https://github.com/fishpepper/pagoda-antenna/files/1083699/pagoda2b.zip

Have fun!
